spanner/spanner_snippets/spanner/spanner_opentelemetry_capture_query_stats_metric.go

```diff
@@ -0,0 +1,107 @@
+//go:build go1.20
```

Comment:
We can merge 3 files into one file and have separate methods for gfe and query stats

Reply:
Done